### PR TITLE
Front End WebPages

### PR DESCRIPTION
**Pull request by tushar1997**
The front end part of our project (that includes Projects page and logout page).

### CODE REVIEW

**Comment by devrishabhgupta** [comment]
@taarakdhingra please review the code

**Comment by devrishabhgupta** [comment]
@tushar1997 add a netlify link so that everyone can see your pages

**Comment by taarakdhingra** [comment]
@devrishabhgupta I'll surely review the code .

File: Frontend/Week 4/Logout.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>PROJECTS</title>
+<link rel="shortcut icon" type="image/png" href="logo 1.png">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"></script>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
```
Comment (devrishabhgupta) [inline Frontend/Week 4/Logout.html:10]: few meta tags missing

File: Frontend/Week 4/index.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+<head>
+<title>PROJECTS</title>
+<link rel="shortcut icon" type="image/png" href="logo 1.png">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"></script>
+    <link href="path/to/select2.min.css" rel="stylesheet" />
+<script src="path/to/select2.min.js"></script>
+    <script>
```
Comment (devrishabhgupta) [inline Frontend/Week 4/index.html:12]: include a separate js file

File: Frontend/Week 4/Logout.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	
+}
+
+.well-lg { margin-left:100px;
+
+               margin-right:100px;
+               background-color: lavender;
+
+              }
+header  { background-color: darkblue;
```
Comment (devrishabhgupta) [inline Frontend/Week 4/Logout.css:25]: try to add a class rather than selecting by tag

**Review by devrishabhgupta — COMMENTED** [review]
There are few points to be noted : -

- i don't have a netlify link to verify your pages
- indentation is missing throughout the code
- use of tag as selector is not preferred

**Comment by devrishabhgupta** [comment]
@tushar1997 since we are short of time I am merging your code

**Pull request merged**